diff --git a/node_modules/react-native-theme-switch-animation/src/index.tsx b/node_modules/react-native-theme-switch-animation/src/index.tsx
index d2175df..b6d37cf 100644
--- a/node_modules/react-native-theme-switch-animation/src/index.tsx
+++ b/node_modules/react-native-theme-switch-animation/src/index.tsx
@@ -54,7 +54,7 @@ const switchTheme = ({
 
 const unfreezeWrapper = () => {
   const defaultRatio = 0.5;
-  setImmediate(() => {
+  setTimeout(() => {
     if (
       localAnimationConfig.type === 'circular' ||
       localAnimationConfig.type === 'inverted-circular'
